doc/source/getting_started/faq.rst

```diff
@@ -12,6 +12,28 @@ Design Language (APDL), Ansys Fluent, and other Ansys products.
 You can use PyAnsys libraries within a Python environment of your choice
 in conjunction with external Python libraries.
 
+What Ansys license do I need to run the Geometry service?
+---------------------------------------------------------
+
+.. note::
+
+   Answered in https://github.com/ansys/pyansys-geometry/discussions/754
+
+The Ansys Geometry service is a headless service developed on top of the
+modelling libraries underneath Discovery and SpaceClaim.
```

Comment:
```suggestion
modeling libraries underneath Discovery and SpaceClaim.
```
The American English preferred spelling is modeling. Could we replace "underneath" with "for"? Having both "on top of" and "underneath" in this one sentence seems to complicate it!

Reply:
Sure, let me do that!